src/lib/Drawer.svelte

```diff
@@ -14,16 +16,34 @@
 	const items = [
 		{
 			icon: BarChartIcon,
-			name: 'Dashboard',
+			name: $_('app.drawer.dashboard'),
 			href: '/dashboard'
 		},
 		{
 			icon: UsersIcon,
-			name: 'Customers',
-			href: '/customers'
+			name: $_('app.drawer.customers'),
+			href: '/customers',
+			expended: false,
+			submenu: [
+				{
+					icon: UsersIcon,
+					name: $_('app.drawer.customers'),
+					href: '/customers'
+				}
+			]
 		}
 	];
 
+	function itemClicked(index: number) {
+		const item = items[index];
+		if (item.submenu) {
+			items[index].expended = !item.expended;
+		}
+		if (item.href) {
+			goto(item.href);
```

Comment:
please, try to use `<ListItem link={item.href}`

Reply:
On ListItem Component

- link prop is not used for navigation
- Here is proof [ListItem Component Source Code](https://github.com/TheComputerM/svelte-materialify/blob/master/packages/svelte-materialify/src/components/List/ListItem.svelte)
- Probably this prop is used just for styling